src/scss/_functions.scss

```diff
@@ -1,7 +1,13 @@
 @function _oHeaderServicesLogo($logo) {
+	// Error if the global $system-code variable is not set.
+	// This is required for image service requests.
+	@if(global-variable-exists('system-code') == false or type-of($system-code) != 'string') {
+		@error 'A global "$system-code" Sass variable must be set to a valid [Bizops system code](https://biz-ops.in.ft.com/list/Systems).';
```

Comment:
How come this is markdown syntax? Does it render correctly in a terminal?

Reply:
It does not... I can remove the square brackets?

Reply:
We could use terminal hyperlinks maybe? https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#quick-example

Reply:
Let's give it a go!